### ISSUE TITLE
An extra vertical scrollbar wrongly appears in the sidebar when it is resized at minimum width, if certain web shops were sent to

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla/side-view#146]
[Affected versions]:
- Firefox 59.0.2 and up
- Side View 0.2.2

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Have a clean Firefox profile with the latest Side View v0.2.2 installed.
- Have the "Side View" sidebar open.

[Steps to reproduce]:
1. Open the Firefox browser with the profile from prerequisites and navigate to https://www.bobomoda.ro/
2. Resize the sidebar to its minimum width and observe the behavior.

[Expected result]:
- The content is correctly rendered no extra scrollbars appear.

[Actual result]:
- An extra vertical scrollbar is displayed.

[Notes]:
- So far I've only encountered this issue on two web shops: the one mentioned above and for certain products pages from https://www.zappos.com (sometimes here the scrollbar appears on the left side).
- Here is a screen recording with this issue: 
![side view - two vertical scrollbars](https://user-images.githubusercontent.com/31244235/38558595-4b3bee80-3cd9-11e8-95dc-4112afce4302.gif)

**Comment by fzzzy** [comment]
I believe this was an artifact of the iframe implementation, which no longer exists. I tested https://www.bobomoda.ro/ and I saw only one scroll bar.

**Comment by Softvision-CristinaBadescu** [comment]
I can still reproduce this issue on "https://www.bobomoda.ro/", using latest Side View v0.3.2 (custom built on 2018-04-13), on latest Nightly build 61.0a1 (2018-04-13), on Windows 10 x64, Mac 10.13.4 and Arch Linux 4.12. I'm not sure if this is related only to "https://www.bobomoda.ro/" website.
However, this issue is no longer on https://www.zappos.com/.

Considering this, I'm going to reopen the issue.

**Comment by fzzzy** [comment]
Yes, I can repro. That's very odd. I don't think this is caused by our implementation but let's leave this open to investigate in the future.

**Comment by fzzzy** [comment]
I'm going to set this as stretch as it is uncommon.

**Comment by mostlygeek** [comment]
Also seeing this problem: 

![an_extra_vertical_scrollbar_wrongly_appears_in_the_sidebar_when_it_is_resized_at_minimum_width__if_certain_web_shops_were_sent_to_ _issue__146_ _mozilla_side-view](https://user-images.githubusercontent.com/83972/40988626-6c4c5c48-68a0-11e8-9c05-311192f8e42a.png)
